usaspending_api/settings.py

```diff
@@ -6,13 +6,17 @@
 import os
 from pathlib import Path
 
-import ddtrace
 import dj_database_url
 from django.db import DEFAULT_DB_ALIAS
 from django.utils.crypto import get_random_string
 
 from usaspending_api.config import CONFIG
 
+TRACE_ENV = "local"
+
+ALLOWED_HOSTS = ["*"]
+ROOT_URLCONF = "usaspending_api.urls"
```

Comment:
Similar question here

Reply:
This was a remnant from when I was configuring otel inside the settings file and the ALLOWED_HOSTS variable and ROOT_URLCONF variable needed to be defined first. But no longer necessary. I undid it.